README.md

```diff
@@ -1 +1,27 @@
-# instagram_own_version
\ No newline at end of file
+# instagram_own_version
+
+A simplified version of Instagram with the ability to add posts,
+set likes or dislikes, follow other people and view your own page.
+
+## Usage
+
+To use this template, first ensure that you have
+[Poetry](https://python-poetry.org/docs/) available.
+
+After that, you should:
+
+1. Install the requirements of the project template by running
+    ```
+    poetry install
+    ```
+2. Activate the virtualenv created by _poetry_:
+    ```
+    poetry shell
+    ```
+
+## Tests
+To run tests make in terminal
+
+```
+    python3 manage.py test
+```
```

Comment:
Let's probably add the details how to run the project locally

Reply:
Ok, should I add information about creating the DB and migrations?

Reply:
Yes. Everything that will help a new person or you since few months/years to run the project.

Reply:
Done